.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "C# (.NET)",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { 
+			// Update 'VARIANT' to pick a .NET Core version: 3.1, 6.0
+			// Append -bullseye or -focal to pin to an OS version.
+			"VARIANT": "6.0-bullseye",
+			// Options
+			"NODE_VERSION": "lts/*"
```

Comment:
I believe this line still needs to be removed.

Reply:
Removed.